contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -5801,13 +5801,19 @@ pltsql_get_object_owner(Oid namespaceId, Oid ownerId)
 
 	Assert(OidIsValid(namespaceId));
 
-	if (sql_dialect != SQL_DIALECT_TSQL || !IS_TDS_CONN())
+	if (sql_dialect != SQL_DIALECT_TSQL || !IS_TDS_CONN() || isTempNamespace(namespaceId))
```

Comment:
we need to verify that normal user (w/o superuser privilege) should not able drop temp schema. Please add such test cases